reference/5-api-publisher/ConfigurationStore/PostgreSql.md

```diff
@@ -0,0 +1,76 @@
+# Configuration Store for PostgreSQL
+
+Enables management of individual connection settings with encryption for securely storing keys and secrets using a supplied password for symmetric key encryption using the `pgcrypto` extension for PostgreSQL.
+
+## Create the Ed-Fi API Publisher Configuration Database and Table
+
+First create the configuration database, as follows:
+```sql
```

Comment:
All code blocks need to have an extra empty line before and after.

Reply:
fixed